packages/extension/src/companion/CompanionMenu.tsx

```diff
@@ -89,14 +89,36 @@ export default function CompanionMenu({
     removeCompanionHelper,
     toggleCompanionExpanded,
   } = useCompanionActions({
-    onBookmarkMutate: () => updatePost({ bookmarked: true }),
-    onRemoveBookmarkMutate: () => updatePost({ bookmarked: false }),
-    onUpvotePostMutate: () => updatePost(mutationHandlers.upvote(post)),
+    onBookmarkMutate: () =>
+      updatePost({
+        update: { bookmarked: true },
+        event: postEventName({ bookmarked: true }),
```

Comment:
Maybe we should convert these to `AnalyticsEvent` as well? Just so it's uniform and just remove `postEventName?

If too much work never mind though.

Reply:
Added in 79cf70a42d4d92aa26471f367d414b17aa0b6afb